tests/kernel/wave/wave_gemm_test.py

```diff
@@ -136,6 +154,11 @@ def repeat(acc: tkl.Register[M, N, tkl.f32]) -> tkl.Register[M, N, tkl.f32]:
             with open(filename, "w") as f:
                 f.write(mb.module_op.get_asm())
 
-        iree_ref = torch.zeros(shape[0], shape[1], dtype=torch.float32)
-        generate_iree_ref("mmt", [a, b], [iree_ref], config)
-        assert_close(c, iree_ref)
+        if run_bench:
+            if dump_perf is not None:
+                config["benchmark_results_file"] = os.path.join(
+                    dump_perf, "iree_" + perf_filename
+                )
+            iree_ref = torch.zeros(shape[0], shape[1], dtype=torch.float32)
+            generate_iree_ref("mmt", [a, b], [iree_ref], config, run_bench=run_bench)
+            assert_close(c, iree_ref)
```

Comment:
This check is needed for correctness, it should present bot in `run_bench=True` and `run_bench=False` paths